runtime/vm/jfr.cpp

```diff
@@ -642,6 +642,7 @@ jint
 initializeJFR(J9JavaVM *vm, BOOLEAN lateInit)
 {
 	PORT_ACCESS_FROM_JAVAVM(vm);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
```

Comment:
This will be unused unless `omrtime_nano_time()` is used instead of `j9time_nano_time()`.

Reply:
Sorry, this is needed for the use of `omrsysinfo_get_number_context_switches()`.